src/main/java/redis/clients/jedis/Protocol.java

```diff
@@ -49,7 +49,7 @@ public final class Protocol {
   private static final String NOPERM_PREFIX = "NOPERM";
 
   private Protocol() {
-    // this prevent the class from instantiation
+    throw new InstantiationError("Must not instantiate this class");
```

Comment:
```    throw new InstantiationError("Do not instantiate this class directly");```

Reply:
99.9999% users won't generate this exception. So the string acts more like a code comment. IMO, as a code comment `Must not ...` is better.

Also, we have this same string in several other places.